SCClassLibrary/Common/Core/Boolean.sc

```diff
@@ -18,9 +18,13 @@ Boolean {
 	keywordWarnings {
 		// turn on/off warnings if a keyword argument is not found
 		_KeywordError
+		^this.primitiveFailed
+	}
+	trace {
+		// this is only available in a special debugging version of the app
```

Comment:
My two cents: We're volunteers. If style is too strict, eventually we will have fewer volunteers. If style is too loose, the codebase becomes a mess. Somewhere in there is the middle way. Noting the position of a comment seems some distance away from that middle, to me at least.

I realize it's a touchy subject, and it makes my flesh crawl when another developer puts spaces near parens or brackets when I wouldn't, or doesn't put spaces near braces when I would, or... or... ... but... we're volunteers.